<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>251</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="layoutFichajes">
       <item>
        <widget class="QPushButton" name="btnResumen">
         <property name="text">
          <string>VER RESUMEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label1">
         <property name="font">
          <font>
           <family>Magneto</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>FICHAJES</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="BtnFichar">
       <property name="text">
        <string>FICHAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnChangePassword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMBIAR CONTRASEÃ‘A</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>160</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="notifications_layout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
